catalog/app/containers/NavBar/Help.js

```diff
@@ -0,0 +1,313 @@
+        title: 'Package total bytes',
+      },
+    ],
+  },
+
+  {
+    namespace: 'Logical operators and grouping',
+    rows: [
+      {
+        example: 'a AND b',
+        id: 'and',
+        syntax: 'AND',
+        title: 'Conjunction',
+      },
+      {
+        example: 'a OR b',
+        id: 'or',
+        syntax: 'OR',
+        title: 'Disjunction',
+      },
+      {
+        example: 'NOT a',
+        id: 'not',
+        syntax: 'NOT',
+        title: 'Negation',
+      },
+      {
+        example: '_exists_: content',
+        id: 'exists',
+        syntax: '_exists_:',
+        title: 'Matches any non-null value for the given field',
+      },
+      {
+        example: '(a OR b)',
+        id: 'group',
+        syntax: '()',
+        title: 'Group terms',
+      },
+    ],
+  },
+
+  {
+    namespace: 'Wildcards and regular expressions',
+    rows: [
+      {
+        example: 'config.y*ml',
+        id: 'asterisk',
+        syntax: '*',
+        title: 'Zero or more characters, avoid leading * (slow)',
+      },
+      {
+        example: 'React.?sx',
+        id: 'question',
+        syntax: '?',
+        title: 'Exactly one character',
+      },
+      {
+        example: '/lmnb[12]/',
+        id: 'regex',
+        syntax: '//',
+        title: 'Regular expression (slows search)',
+      },
+    ],
+  },
+]
+
+function Code({ children }) {
+  const classes = useCodeStyles()
+
+  return <code className={classes.root}>{children}</code>
+}
+
+function Item({ item }) {
+  const t = M.useTheme()
+  const sm = M.useMediaQuery(t.breakpoints.up('sm'))
+
+  const { example, syntax, title } = item
+  return (
+    <M.Grid container>
+      <M.Grid item xs={6} sm={3}>
+        <M.Typography variant="body2">
+          <Code>{syntax}</Code>
+        </M.Typography>
+      </M.Grid>
+      <M.Grid item xs>
+        <M.Typography variant="body2">{title}</M.Typography>
+      </M.Grid>
+      {sm && (
+        <M.Grid item xs={4}>
+          <M.Typography variant="body2">
+            <Code>{example}</Code>
+          </M.Typography>
+        </M.Grid>
+      )}
+    </M.Grid>
+  )
+}
+
+function ItemsHeader() {
+  const t = M.useTheme()
+  const sm = M.useMediaQuery(t.breakpoints.up('sm'))
+
+  const classes = useItemsHeaderStyles()
+
+  return (
+    <M.Grid container className={classes.root}>
+      <M.Grid item xs={6} sm={3}>
+        <M.Typography variant="subtitle2">Command</M.Typography>
+      </M.Grid>
+      <M.Grid item xs>
+        <M.Typography variant="subtitle2">Description</M.Typography>
+      </M.Grid>
+      {sm && (
+        <M.Grid item xs sm={4}>
+          <M.Typography variant="subtitle2">Example</M.Typography>
+        </M.Grid>
+      )}
+    </M.Grid>
+  )
+}
+
+function DocsExternalLink() {
+  const classes = useDocsExternalLinkStyles()
+
+  return (
+    <M.Typography variant="body2" component="p" className={classes.root}>
+      Quilt uses ElasticSearch 6.7 query string queries.{' '}
```

Comment:
i think it makes sense to use the `ES_V` constant here to have a single source of truth